README.md

```diff
@@ -1,15 +1,60 @@
-# tket2
+# tket2: The Hardware Agnostic Quantum Compiler
 
 [![build_status][]](https://github.com/CQCL-DEV/tket2/actions)
 ![msrv][]
 [![codecov][]](https://codecov.io/gh/CQCL/tket2)
 
-Version 2 of the TKET compiler.
-
   [build_status]: https://github.com/CQCL-DEV/hugr/workflows/Continuous%20integration/badge.svg?branch=main
   [msrv]: https://img.shields.io/badge/rust-1.75.0%2B-blue.svg
   [codecov]: https://img.shields.io/codecov/c/gh/CQCL/tket2?logo=codecov
 
+TKET2 is an open source quantum compiler developed by Quantinuum. Central to
```

Comment:
It is a bit of a shame to repeat these words exactly from the rust crate docs...

Reply:
These two are read from different places. The crate docs only really appear in `docs.rs`, whereas the readme is for github/crates.io.

We could make the crate docs more technical at some point, this is just so we have _something_ useful here.

Reply:
Yes, I mean, it's a shame not to be able to link between them / copy text (they seem to be aimed at very similar audiences), but I admit I don't see a good way to do that, so ok nvm

Reply:
Oh, should possibly be `Quantinuum Ltd`